WalletWasabi/Blockchain/Analysis/FeesEstimation/FeeProviders.cs

```diff
@@ -49,20 +36,23 @@ private void SetAllFeeEstimate()
 		{
 			lock (Lock)
 			{
+				AllFeeEstimate? feeEstimateToSet = null;
 				IFeeProvider[] providerArray = Providers.ToArray();
 				for (int i = 0; i < providerArray.Length - 1; i++)
 				{
 					IFeeProvider provider = providerArray[i];
 					var af = provider.AllFeeEstimate;
 					if (af is { } && af.IsAccurate)
 					{
-						AllFeeEstimate = af;
-						return;
+						feeEstimateToSet = af;
```

Comment:
new syntax can be used here:

```csharp
if (provider.AllFeeEstimate is { } af && af.IsAccurate)
{
    feeEstimateToSet = af;
    break;
}
```

Reply:
ACK

Reply:
I think the code that @kiminuo suggested is shorted, easier to read, and you don't have to create another variable (`isAccurate`)